applications/fax/src/fax_smtp.erl

```diff
@@ -892,3 +893,14 @@ write_tmp_file(Filename, Extension, Body) ->
             lager:debug("error writing file ~s : ~p", [Filename, Error]),
             Error
     end.
+
+send_outbound_smtp_fax_error(State) ->
+    Message = props:filter_empty(
+                [{<<"Account-ID">>, State#state.account_id}
+                ,{<<"Fax-From-Email">>, State#state.from}
+                ,{<<"Fax-To-Email">>, State#state.to}
+                ,{<<"Errors">>, State#state.errors}
```

Comment:
How about including more information?
peer_ip, proxy, numbers, ... would this help?

@lazedo ideas?